config/importmap.rb

```diff
@@ -1,12 +1,16 @@
 # Pin npm packages by running ./bin/importmap
 
 pin "application", preload: true
+pin "jquery" # @3.7.1
+pin "@rails/ujs", to: "@rails--ujs.js" # @7.1.3
+# stimulus.min.js is a compiled asset from stimulus-rails gem
+pin "@hotwired/stimulus", to: "stimulus.min.js"
+# stimulus-loading.js is a compiled asset only available from stimulus-rails gem
+pin "@hotwired/stimulus-loading", to: "stimulus-loading.js"
+pin_all_from "app/javascript/controllers", under: "controllers"
+pin "clipboard", preload: false # @2.0.11
```

Comment:
Why the change to preload false?

Reply:
These are only used on specific pages. Seemed aggressive to preload them. I think this change snuck in with my overall changes though and wasn't really intended.

Do you think I should revert this?

Reply:
It's fine so long as it's intentional and you've tested that it doesn't break the clipboard functionality

Reply:
I'll revert for now. I want to cleanup the loading but I'll do it in a separate PR.